build_runner/CHANGELOG.md

```diff
@@ -8,17 +8,16 @@
 
 - Upgrade to `frontend_service_client` version 3.
 
-## 2.3.0
-
+## 2.3.3-dev
```

Comment:
this should remain as it was, there is a section higher up already for 2.3.3-dev